siesta-jackson/src/main/java/org/sonatype/sisu/siesta/jackson/ObjectMapperProvider.java

```diff
@@ -40,8 +39,7 @@ public ObjectMapperProvider() {
         .with(new JacksonAnnotationIntrospector());
     // do not write null values
```

Comment:
This comment should be moved/removed as it no longer applies to the line below it.